packages/venia-ui/lib/queries/getCartDetails.graphql

```diff
@@ -1,5 +1,6 @@
 query getCartDetails($cartId: String!) {
     cart(cart_id: $cartId) {
+        id
```

Comment:
This was missing, and potentially might cause cache misses.

However, in order for the trigger to be updated after mutations like `removeItem` we would have to start using named fragments inside the remove item mutation.

Reply:
These cart queries use different versions of schema, so with the inclusion of id heuristic fragment matching wouldn't hit (must be exact). Even if we switched to consuming fragments here, we'd have to update the components to the new schema, which is making the scope creepy.

**If** this was a priority, I would probably just do a manual cache update in our mutations.

Reply:
As we discovered just now this works because Apollo sees "partial data" and thus makes a network call.